packages/plugin-legacy/README.md

```diff
@@ -158,6 +158,14 @@ These values can also be retrived via
 const { cspHashes } = require('@vitejs/plugin-legacy')
 ```
 
+## Inline Scripts
+It may be desirable to recreate inline scripts generated by the legacy plugin, e.g. if Vite is not being used to generate your HTML.
+
+These values can be retrieved via
```

Comment:
Just some formatting / blank lines

```suggestion

It may be desirable to recreate inline scripts generated by the legacy plugin, e.g. if Vite is not being used to generate your HTML.

These values can be retrieved via

```